MaxText/pyconfig.py

```diff
@@ -76,7 +76,7 @@ def validate_no_keys_overwritten_twice(keys1: list[str], keys2: list[str]):
   overwritten_keys = [k for k in keys1 if k in keys2]
   if overwritten_keys:
     raise ValueError(
-        f"Keys {overwritten_keys} are overwritten from both the model" 
+        f"Keys {overwritten_keys} are overwritten from both the model"
```

Comment:
Please avoid all whitespace changes in the future.

Reply:
Oh just saw this. This is automatic changes from Code Server with white space trailing setup.